code/modules/spacepods/spacepod.dm

```diff
@@ -812,11 +812,17 @@ obj/spacepod/proc/add_equipment(mob/user, var/obj/item/spacepod_equipment/SPE, v
 	if(!istype(user))
 		return
 
-	if(usr.incapacitated()) // unconscious and restrained people can't let themselves out
+	if(usr.stat) // unconscious people can't let themselves out
 		return
 
 	occupant_sanity_check()
 
+	if(usr.restrained())
+		to_chat(usr, "<span class='notice'>You attempt to stumble out of the [src]. This will take two minutes.</span>")
```

Comment:
This should give some message to the pilot at least, so they know if their prisoner is trying to break out